data/test-gsym.ld

```diff
@@ -12,6 +12,27 @@ SECTIONS {
     *(.bss)
   }
 
+  /* DWARF debug sections.
+     Symbols in the DWARF debugging sections are relative to the beginning
+     of the section so we begin them at 0.  */
```

Comment:
styling nit: usually for multi-line comments `*/` is put on a separate line at the end

Reply:
It's verbatim what the default linker script on my system contains, but sure, can change it.